crates/dbsp/benches/gdelt/data.rs

```diff
@@ -234,11 +234,12 @@ pub fn parse_personal_network_gkg(
                             if let Some(normals) = normalizations.get(person) {
                                 people.extend(normals.iter().cloned());
                             } else {
-                                people.push(
-                                    interner
-                                        .get_or_insert_with(person, |person| ArcStr::from(person))
-                                        .clone(),
-                                );
+                                people.push(loop {
```

Comment:
Why'd you change this?

Reply:
`get_or_insert_with` is nightly-only in `std`.

I switched from `hashbrown` to standard collections because `hashbrown` only supports `rkyv` as of its latest release (0.14.0) but we have dependencies that pin `rkyv` to earlier versions.

Reply:
Ok, could you get rid of the `loop` stuff then? It's kinda funky and insert returns a reference to the inserted value so we can just clone & return that

Reply:
HashSet insert just returns a bool: https://doc.rust-lang.org/stable/std/collections/hash_set/struct.HashSet.html#method.insert